src/dashboard/Data/Browser/AddColumnDialog.react.js

```diff
@@ -74,21 +193,42 @@ export default class AddColumnDialog extends React.Component {
         cancelText={'Never mind, don\u2019t.'}
         onCancel={this.props.onCancel}
         onConfirm={() => {
-          this.props.onConfirm(this.state.type, this.state.name, this.state.target);
+          this.props.onConfirm(this.state);
         }}>
         <Field
           label={
             <Label
               text='What type of data do you want to store?' />
-            }
+          }
           input={typeDropdown} />
         {this.state.type === 'Pointer' || this.state.type === 'Relation' ?
           <Field
             label={<Label text='Target class' />}
             input={this.renderClassDropdown()} /> : null}
         <Field
           label={<Label text='What should we call it?' description={'Don\u2019t use any special characters, and start your name with a letter.'} />}
-          input={<TextInput placeholder='Give it a good name...' value={this.state.name} onChange={(name) => this.setState({ name })} />}/>
+          input={<TextInput placeholder='Give it a good name...' value={this.state.name} onChange={(name) => this.setState({ name })} />} />
+        {
+          /*
+            Allow include require fields and default values if the parse-server
+            version is greater than or equal 3.7.0, that is the minimum version
+            support this feature and check if the field is not a relation
+          */
+          semver.valid(this.props.parseServerVersion) &&
+          semver.gte(this.props.parseServerVersion, '3.7.0') &&
+          this.state.type !== 'Relation' ?
+            <>
+              <Field
+                label={<Label text='What is the default value?' description='If no value is specified for this column, it will be filled with its default value' />}
```

Comment:
```suggestion
                label={<Label text='What is the default value?' description='If no value is specified for this column, it will be filled with its default value.' />}
```